tests/testthat/test-oc_check_query.R

```diff
@@ -257,3 +257,17 @@ test_that("oc_check_logical checks no_record", {
     )
   )
 })
+
+# Test oc_check_between() -------------------------------------------------
+test_that("oc_check_logical checks no_record", {
+  # numeric()
+  symbol <- 11
+  expect_error(oc_check_between(symbol, 0, 10))
```

Comment:
you could use `expect_snapshot_error()`.

Reply:
using `expect_snapshot(error = TRUE)` now, so the snap file also contains the code and not only the error message (which it does when using `expect_snapshot_error()`